test/functionalTest/cases_xml/0789_notifyFormat/notifyFormat_ngsi10.test

```diff
@@ -1,588 +0,0 @@
-# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I understand that all .test under 0789_notifyFormat are testing the `notifyFormat` feature to override default format, which is not longer supported as we have only one format (JSON).